assets/js/components/KeyMetrics/key-metrics-widgets.js

```diff
@@ -412,6 +412,7 @@ const KEY_METRICS_WIDGETS = {
 			'google-site-kit'
 		),
 		requiredConversionEventName: [ 'add_to_cart' ],
+		displayInList: shouldDisplayWidgetWithConversionEvent,
```

Comment:
@10upsimon, I suppose the idea here is to ensure that this widget doesn't appear when the `conversionReporting` feature flag is not enabled for the site, right? The problem is that the `shouldDisplayWidgetWithCustomDimensions` function doesn't check for that feature flag, hence this widget will continue appearing. Or I miss something?

Reply:
@eugene-manuilov the very first thing the function checks for is the feature flag?

```
	if ( ! isFeatureEnabled( 'conversionReporting' ) ) {
		return false;
	}
```

Reply:
@eugene-manuilov That function checks for feture flag before checking for detected events https://github.com/google/site-kit-wp/blob/0525c3b6f74dd88d924de35ef7658ad67ce4e774/assets/js/components/KeyMetrics/shouldDisplayWidgetWithConversionEvent.js#L25-L27

Reply:
Yeah, you are right guys. I looked at the wrong function. Looks good now.